dbt/models/reporting/schema.yml

```diff
@@ -126,14 +126,32 @@ models:
           combination_of_columns:
             - year
             - stage
-            - municipality_name
+            - municipality
             - class
       - expression_is_true:
           name: reporting_vw_assessment_roll_muni_no_nulls
           expression: |
             stage IS NOT NULL
-            AND municipality_name IS NOT NULL
+            AND municipality IS NOT NULL
             AND class IS NOT NULL
+      - expression_is_true:
```

Comment:
These new tests should hopefully be enough to ensure the 3 new columns are properly constructed.